scripts/jenkins/Jenkinsfile

```diff
@@ -1523,6 +1528,55 @@ def publishDebianPackages(remote="a7") {
   )
 }
 
+
+/* Publishes all files necessary for hosting a bionic package
+ * @param remote where the repository is located
+ */
+def publishBionicPackages(remote="doc.libelektra.org") {
+  // This path must coincide with the incoming dir on community
+  def remotedir = '/srv/repositories/incoming/'
```

Comment:
If I did not overlook something, the different remotedir is the only difference to the function publishDebianPackages. Simply make remotedir also an argument, then you can call publishPackages for both debian and bionic (and other versions that hopefully come in the future).